lib/core/src/lib/layout/components/header/header.component.html

```diff
@@ -5,15 +5,16 @@
         *ngIf="showSidenavToggle && position === 'start'"
         id="adf-sidebar-toggle-start"
         data-automation-id="adf-menu-icon"
-        class="mat-icon-button adf-menu-icon"
+        class="mat-icon-button adf-menu-icon adf-header-icon-button"
```

Comment:
is it possible to also remove the 'mat-icon-button' class? as this one should not be used there